src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -777,42 +793,44 @@ def get_rom_directory(self, rom_name):
         self._log_key = "GetRomDirectory"
 
         if not self.evaluation_is_initialized:
-            msg = "TwinModel has not been initialized! "
-            msg += "Please initialize evaluation before to call this method!"
+            msg = "Twin model has not been initialized. "
+            msg += "Initialize evaluation before calling this method."
             self._raise_error(msg)
 
         if self.tbrom_info is None:
-            self._raise_error("Twin model does not include any TBROM!")
+            self._raise_error("Twin model does not include any TBROM.")
 
         if rom_name not in self.tbrom_names:
-            msg = f"The provided rom_name {rom_name} has not been found in the available TBROM names. "
-            msg += f"Please call this method with a valid TBROM name."
-            msg += f"\n Available TBROM name are: {self.tbrom_names}"
+            msg = f"The provided ROM name {rom_name} has not been found in the available TBROM names. "
+            msg += f"Call this method with a valid TBROM name."
+            msg += f"\n Available TBROM names are: {self.tbrom_names}."
             self._raise_error(msg)
 
         return os.path.join(self.tbrom_directory_path, rom_name)
 
     def get_snapshot_filepath(self, rom_name: str, evaluation_time: float = 0.0):
         """
-        Get the snapshot file associated to a Reduced Order Model (ROM) available in the TwinModel and evaluated at the
-        given time instant. The snapshot file contains the field results of the ROM.
+        Get the snapshot file associated with a ROM (reduced order model) in the twin model and evaluated at the
+        given time instant.
+
+        The snapshot file contains the field results of the ROM.
 
         Parameters
         ----------
         rom_name : str
-            This is the name of a ROM model that is available in the TwinModel. See TwinModel.tbrom_names property to
-            get a list of available ROM model.
-        evaluation_time: float
-            This is the evaluation time at which you want to get the snapshot file. The method returns None if no
-            snapshot file is available at this evaluation_time. Two evaluation times can be distinguished up to 6 digits
-            after the comma.
+            Name of the ROM. To get a list of available ROMs, see the
+            :attr:`pytwin.TwinModel.tbrom_names` attribute.
+        evaluation_time: float, optional
+            Evaluation time at which to get the snapshot file. The default is ``0.00`. If no
+            snapshot file is available at this evaluation time, the method returns ``None``.
+            Two evaluation times can be distinguished up to six digits after the comma.
```

Comment:
Again, I don't understand what this sentence means.

Reply:
@lboucin  I like your original first sentence as is. As indicated earlier, it's the last sentence that confuses me.